arcgis-ios-sdk-samples/Layers/Query with CQL filters/QueryWithCQLFiltersSettingsViewController.swift

```diff
@@ -0,0 +1,176 @@
+// Copyright 2021 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+protocol QueryWithCQLFiltersSettingsViewControllerDelegate: AnyObject {
+    func settingsViewController(_ controller: QueryWithCQLFiltersSettingsViewController, queryParameters: AGSQueryParameters)
+}
+
+class QueryWithCQLFiltersSettingsViewController: UITableViewController {
+    // MARK: Storyboard views
+    
+    /// The cell for where clause options.
+    @IBOutlet var whereClauseCell: UITableViewCell!
+    /// The cell for max features input.
+    @IBOutlet var maxFeaturesCell: UITableViewCell! {
+        didSet {
+            maxFeaturesCell.detailTextLabel?.text = String(maxFeatures)
+        }
+    }
+    /// The start date picker.
+    @IBOutlet var startDatePicker: UIDatePicker!
+    /// The end date picker.
+    @IBOutlet var endDatePicker: UIDatePicker!
+    /// The switch to control if date pickers are enabled.
+    @IBOutlet var dateFilterSwitch: UISwitch!
+    
+    // MARK: Properties
+    
+    /// A delegate to notify other view controllers.
+    weak var delegate: QueryWithCQLFiltersSettingsViewControllerDelegate?
+    
+    let numberFormatter = NumberFormatter()
+    let sampleWhereClauses = [
+        // Sample Query 1: Features with an F_CODE property of "AP010".
+        // A cql-text query.
+        "F_CODE = 'AP010'",
+        // A cql-json query.
+        "{ \"eq\" : [ { \"property\" : \"F_CODE\" }, \"AP010\" ] }",
+        // Sample Query 2: Features with an F_CODE property matching the pattern.
+        "F_CODE LIKE 'AQ%'",
+        // Sample Query 3: Use cql-json to combine "before" and "eq" operators
+        // with the logical "and" operator.
+        "{\"and\":[{\"eq\":[{ \"property\" : \"F_CODE\" }, \"AP010\"]},{ \"before\":[{ \"property\" : \"ZI001_SDV\"},\"2013-01-01\"]}]}"
```

Comment:
I thought Zack said in the review to leave the json out. Did I misunderstand?

Reply:
He said to leave the *second* json out, which was the one with `like` that I've removed.